### PR TITLE
Share of daily rapid tests. One empirical dataset. Introduce summer simulation period. Add work scenarios.

### PR DESCRIPTION
**Pull request by roecla**
- [x] save and plot the share of daily rapid tests
- [x] create one empirical data set with all empirical time series
- [x] use empirical data set everywhere
- [ ] ~~switch to daily incidence per 1 mio inhabitants instead of weekly per 100 000~~
- [ ] ~~add random rapid test scenarios~~
- [x] add other plots for the sid-paper draft
- [x] save more data from plots as tables
- [x] add 3rd simulation period "summer"
- [x] add home office and work testing scenarios and simulate their impact over the summer  
- [x] add effective R by the RKI to the empirical data.

### CODE REVIEW

File: src/plotting/plotting.py [hunk 1]
```diff
@@ -35,6 +32,20 @@
 ORDERED_COLORS = [BLUE, TEAL, YELLOW, ORANGE, RED, PURPLE]
 
 
+OUTCOME_TO_EMPIRICAL_LABEL = {
+    "new_known_case": "official case numbers",
+    "newly_deceased": "official case numbers",
+    "share_ever_rapid_test": "share of Germans reporting to have\n"
+    + "ever done a rapid test",
```
Comment (tobiasraabe) [inline src/plotting/plotting.py:39]:
```suggestion
    "ever done a rapid test",
```

File: src/policies/single_policy_functions.py [hunk 2]
```diff
@@ -87,11 +87,13 @@ def reduce_work_model(
...
-    threshold = 1 - attend_multiplier
-    if isinstance(threshold, pd.Series):
-        threshold = states["state"].map(threshold.get)
+    threshold_short = 1 - attend_multiplier
+    if isinstance(threshold_short, pd.Series):
+        threshold = states["state"].map(threshold_short.get)
         # this assert could be skipped because we check in
         # task_check_initial_states that the federal state names overlap.
+        if threshold.dtype == "category":
+            breakpoint()
```
Comment (tobiasraabe) [inline src/policies/single_policy_functions.py]: Raise an error if it is not accidentally committed
```suggestion
```

File: src/prepare_data/task_prepare_vaccination_data.py [hunk 3]
```diff
@@ -122,7 +123,7 @@ def _clean_vaccination_data(df):
 
 
 def _plot_series(sr, title, label=None):
-    fig, ax = plt.subplots(figsize=(15, 5))
+    fig, ax = plt.subplots(figsize=PLOT_SIZE)
```
Comment (tobiasraabe) [inline src/prepare_data/task_prepare_vaccination_data.py:126]: You could also set in the `matplotlibrc` under `figure.figsize: 12, 8` for all newer plots.

File: src/simulation/task_run_simulation.py [hunk 4]
```diff
@@ -48,10 +49,10 @@ def _create_simulation_parametrization():
                 specs["params_scenario"],
                 specs["start_date"],
                 specs["end_date"],
-                specs.get("save_last_states", False),
+                save_last_states,
                 produces,
                 500 + 100_000 * seed,
-                is_resumed,
+                is_resumed is not False,
```
Comment (tobiasraabe) [inline src/simulation/task_run_simulation.py:55]:
```suggestion
                is_resumed,
```

**Review by tobiasraabe — APPROVED** [review]
All good! 👍

File: src/prepare_data/task_prepare_vaccination_data.py (hunk 3 above)
Comment (roecla) [reply src/prepare_data/task_prepare_vaccination_data.py:126]: Not now but will do that later. Thanks for the pointer!

File: src/plotting/task_plot_rki_incidences.py [hunk 5]
```diff
@@ -14,7 +15,7 @@
 @pytask.mark.depends_on(
     {
         "rki": BLD / "data" / "processed_time_series" / "rki.pkl",
-        "plotting.py": SRC / "plotting" / "plotting.py",
+        # "plotting.py": SRC / "plotting" / "plotting.py",
```
Comment (janosg) [inline src/plotting/task_plot_rki_incidences.py]:
```suggestion
```

File: src/plotting/task_create_scenario_comparison_tables.py [hunk 6]
```diff
@@ -25,7 +25,7 @@ def _create_deps(scenarios, groupby):
     py_dependencies = {
         "config.py": SRC / "config.py",
         "scenario_config.py": SRC / "simulation" / "scenario_config.py",
-        "plotting.py": SRC / "plotting" / "plotting.py",
+        # "plotting.py": SRC / "plotting" / "plotting.py",
```
Comment (janosg) [inline src/plotting/task_create_scenario_comparison_tables.py:28]: We should delete this or comment it in; Same in other places

**Review by janosg — APPROVED** [review]

File: src/plotting/task_create_scenario_comparison_tables.py (hunk 6 above)
Comment (roecla) [reply src/plotting/task_create_scenario_comparison_tables.py]: Added them back in. It worked better to use `pytask.mark.persist` anyways.

**Pull request merged**